### ISSUE TITLE
BoldAsFont not honored when text is colored with ANSI SGR 90-97

### ISSUE THREAD

**Issue by nu774** [mintty/mintty#936]
I mean, 
```
 echo -e "\e[95m\e[1mfoo"
```
is rendered in bold font even when BoldAsFont is no, and I don't want it.

### Background:
I regularly launch cmd.exe, wsl.exe and the like from cygwin mintty bash shell. Thanks to the recent ConPty support by cygwin, it just works.
Bad thing is that  ConPty seems to always add color escape sequence (ANSI SGR 90-97) like the above example even when the original process just outputs bold sequence (ANSI SGR 1).
As a result, bold texts came from ConPty is always rendered in bold font by mintty, which is not what I want.

**Comment by avih** [comment]
This is what the manual says about bold:
```
       Note: Bold Behaviour
              When the bold text attribute is set, mintty distinguishes three classes
              of colours:
              – Default: The default terminal foreground colour.
              – ANSI-8: The ANSI colours 0..7 (used for SGR 30..37).
              – Extended: True colours and the rest of the 256 colours palette.

              The colour classes are affected by the bold text attribute as follows:
              – Extended colours are always shown with a boldened font only.
              – When both BoldAsFont and BoldAsColour are disabled, mintty engages a
              mode similar to the xterm default behaviour: ANSI-8 is displayed with a
              bold font and a different colour, Default colour only uses a bold font.
              – Otherwise, Default and ANSI-8 colours are affected by BoldAsFont and
              BoldAsColour independently, such that it is possible to choose only
              bold font, or only different colour, or both.
              – Note that Default bold display can be overridden by the BoldColour
              mode.
```

So your case is a bright color (SGR 95) which is then made bold (SGR 1). So yes, currently mintty will embolden the in for this case.

The current behavior is that mintty will change the appearance in _some_ way to ensure bold looks different than non bold. This is not something which you can currently override or disable.

As the docs say, BoldAsFont only ever affects ANSI-8 and sometimes the default color. The rest get emboldened regardless.

**Comment by mintty** [comment]
You can disable the bold attribute altogether, setting `SuppressSGR=1`, if that's what you want.

**Comment by nu774** [comment]
SupressSGR=1 did the trick.
Thanks.

**Comment by mintty** [comment]
Released 3.1.0.
